src/App.css

```diff
@@ -1,39 +1,8 @@
-.App {
-  text-align: center;
+body {
+  background-color: rgb(217, 239, 255);
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+#root {
```

Comment:
- [ ] Please don't use id as a selector. you can use classes instead.

Reply:
Resolved